client/app/views/tags.coffee

```diff
@@ -17,7 +15,7 @@ module.exports = class TagsView extends BaseView
         return this
 
     tagAdded: (ev, ui) ->
-        ui.tag.css 'background-color', colorHash ui.tagLabel
+        ui.tag.css 'background-color', ColorHash.getColoru(ui.tagLabel, 'cozy')
```

Comment:
I'm soon going to Asia, that's why I add it (if you don't understand why, I'll explain ;)). I fix it asap.